README.md

```diff
@@ -19,9 +19,11 @@
 - [📖 Usage and Documentation](#usage-and-documentation)
 - [💎 Modules](#-modules)
 - [Consuming library with CMake](#consuming-library-with-cmake-cmake-322-or-newer)
-- [⚒️ Compiler support](#compiler-support)
+- [⚒️ Compiler support](#⚒️-compiler-support)
```

Comment:
Ops! it should be `[⚒️ Compiler support](#compiler-support)` instead. The README link is broken now: https://github.com/cieslarmichal/faker-cxx?tab=readme-ov-file#table-of-contents

Reply:
I fixed that directly on main now